packages/py-api/py_api/middleware/exception_handler.py

```diff
@@ -30,3 +29,7 @@ async def general_exception_handler(request: Request, exc: Exception) -> JSONRes
                 status_code=500,
                 content=content,
             )
+
+        @app.exception_handler(InvalidId)
+        async def handle_invalid_object_id(request: Request, exc: InvalidId) -> JSONResponse:
```

Comment:
This is under discussion over chat

Reply:
I will create a separate issue for this, so merge it as it is, because many new tickets depend on this ticket.